Update README.md

tiduscrying opened the pull request:
added rough instructions on how to properly load cdtracks and mods per request on GBATemp.

Rinnegatamante commented:
There are some typo errors and formattation errors.

Rinnegatamante commented:
There's also formatting issues still with bolding or maybe that's just GitHub text viewer that fucks up formatting, can't really say.

tiduscrying commented:
Alright. 

It's -> its
Changed all instances of "Hipnotic" and "Rogue" to their official expansion pack names
Foler -> folder
Removed ogg format requirement. Misunderstanding. 
There is nothing wrong with saying "has support for," but I changed it to "supports."
Offical -> official
Reworded to "expansions"
compatable -> compatible
According to github's markdown formatting cheat sheet, my bolding was correct, but I forgot to close it with another two asterisks at the end. 

I sincerely appreciate the work you've done on the project. My typos, mistakes and other inconsistencies are my apology. People on GBATemp and Wololo had questions on how to get mods and CD audio working, and I figured adding it to the readme would be beneficial to everyone. I am just trying to help, that is all.
